src/js/lib/sheettodom.js

```diff
@@ -0,0 +1,25 @@
+import sheetUrl from './sheetURL'
+import reqwest from 'reqwest'
+
+
+export default function sheettodom(sheetID, sheetName, el, callback) {
+
+    var sheet = sheetUrl(sheetID);
+
+    reqwest({
+        'url': sheet,
+        'type': 'json',
+        'crossOrigin': true,
+        'success': resp => {
+
+            //get list of elements with data-sheet-attribute
+            for (const node of el.querySelectorAll('[data-sheet-attribute]')) {
```

Comment:
Unfortunately this does not work on a nodelist. See https://developer.mozilla.org/en/docs/Web/API/NodeList#Why_is_NodeList_not_an_Array 
I got this style of iterator from: https://jakearchibald.com/2014/iterators-gonna-iterate/